README.md

```diff
@@ -306,9 +306,30 @@ This directory contains proofs of interesting homotopy-theoretic theorems.
 
 ### Cohomology (directory `theorems/cohomology/`)
 
-This directory contains proofs of interesting cohomology-theoretic theorems.
-
-TODO: describe more precisely each file
+This directory contains proofs of interesting cohomology-theoretic theorems. Many results in this directory are described in [Evan Cavallo's thesis][ecavallo-thesis].
+
+- `Bouquet`: Shows that the cohomology in degree n of a bouquet of n-spheres indexed by a type `I`, which has choice, is isomorphic to the product of `I` copies of `C2 0` (the 0-th cohomology of the 0-sphere) for an ordinary cohomology theory.
+- `ChainComplex`: Defines the data types of (co)chain complexes and equivalences between them, defines their (co)homology groups and proves that equivalences between complex induce equivalences between their cohomology groups.
+- `CoHSpace`: Contains simple lemmas about the cohomology of co-H-spaces.
+- `Cogroup`: Given a type `X` with a cogroup structure and a type `Y`, proves that the map `(X ⊙→ Y) → (C n Y →ᴳ C n X)` is a group homomorphism for any cohomology theory `C`.
+- `Coproduct`: Proves that `C n (X ⊙⊔ Y) ≃ᴳ C n (X ⊙∨ Y) ×ᴳ C2 n` (where `C2 n` is the `n`-th cohomology of the 0-sphere) for any cohomology theory `C`.
+- `DisjointlyPointedSet`: Shows that the cohomology of a separable pointed set `X`, which has choice, is the `MinusPoint X`-fold product of `C2 0` (the 0-th cohomology of the 0-sphere) in degree 0 and trivial in higher degrees for any ordinary cohomology theory `C`.
+- `EMModel`: Constructs the Eilenberg–MacLane spectrum given an abelian group and shows that its induced cohomology theory is ordinary.
+- `InverseInSusp`: Shows that the homomorphism Cⁿ(ΣX) → Cⁿ(ΣX) mapping an element to its inverse is induced by a map ΣX → ΣX.
+- `LongExactSequence`: Given a map `f : X → Y`, constructs the sequence Cⁿ(Y) → Cⁿ(X) → Cⁿ⁺¹(cofib(f)) → Cⁿ⁺¹(Y) → ⋯ and shows that it is exact.
+- `MayerVietoris`: Given a pointed span X ←f– Z –g→ Y, shows the cofiber space of the natural map `reglue` : X ∨ Y → X ⊔_Z Y is equivalent to the suspension of Z. Using this equivalence one can derive the Mayer–Vietoris sequence from the long exact sequence associated with `reglue`.
```

Comment:
Hmm. Somehow GitHub-markdown is unhappy about the underscore in `⊔_Z`. Could you escape it?

Reply:
fixed